distr_tracing/distr_tracing_otel/distr-tracing-otel-forwarding.adoc

```diff
@@ -0,0 +1,136 @@
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-forwarding"]
+= Forwarding traces to TempoStack by using OpenTelemetry Collector
+include::_attributes/common-attributes.adoc[]
+:context: distr-tracing-otel-forwarding
+
+To configure log forwarding to the TempoStack, you deploy and configure an OpenTelemetry collector.
+
+.Prerequisites
+
+* Tempo Operator is installed.
+* TempoStack is deployed on the cluster.
+* Red Hat OpenShift distributed tracing data collection operator is installed.
```

Comment:
@rubenvp8510, I suggest removing the prerequisites section completely because these prerequisites are self-explanatory, which means we don't have to mention them: 
```suggestion
.Prerequisites

* Tempo Operator is installed.
* TempoStack is deployed on the cluster.
* Red Hat OpenShift distributed tracing data collection operator is installed.
```